docs/docs/installation/index.md

```diff
@@ -195,6 +195,25 @@ To implement this:
 * Paste those lines into your `values.yaml` file
   and modify their values in that file.
 
+### Installing on Openshift
```

Comment:
I think this should be a separate page in the "Configuration" section.

Reply:
Thank you, I'll move this tonight.

Reply:
I moved it to [docs/docs/installation/configuration/Openshift.md](https://github.com/smuda/keptn-lifecycle-toolkit/blob/feature/make-chart-openshift-compliant/docs/docs/installation/configuration/Openshift.md)